pkg/controller/redirect/index.go

```diff
@@ -34,8 +35,15 @@ func (c *Controller) HandleIndex() http.Handler {
 
 		for hostname, region := range c.hostnameToRegion {
 			if host == hostname {
-				sendTo := fmt.Sprintf("ens:/%s&r=%s", path, region)
```

Comment:
FWIW, this was "okay" in the sense that path has a leading slash.
But building the URI explicitly is better.